### DISCUSSION TITLE
System's "standard" browser instead "Chrome based browser" to open links in view mode of a Markdown document?

### DISCUSSION THREAD

**Discussion by mglh** [gsantner/markor#1520] [Q&A] [answered]
When opening a link in the view mode of a Markdown document, it is opened in the "Chrome based browser" that is part of Markor.
Is it possible to use the system's "standard" browser at this point, e.g. Firefox?

**Comment by gsantner** [comment, marked as answer]
Since Markor v2.8.4 there is a settings option "Chrome Custom Tabs". It controls exactly what you ask for.

![screenshot-2021-12-04__03-36-22](https://user-images.githubusercontent.com/6735650/144693963-332a6357-14b6-464d-b067-85f466d33974.png)

**Reply by mglh** [reply]
Gregor, thanks =:)
Since I'm using F-Droid.org on my device, I'm still at v2.8.3 at the moment.
I'm looking forward to see v.2.8.4 appear.

**Comment by mglh** [comment]
My device is on v2.8.4 now, and the option you've mentioned is set.
However, links are still opened in the "Chrome-based browser".
Am I missing something?

**Reply by gsantner** [reply]
uncheck to disable.

**Reply by mglh** [reply]
Gregor, thanks again =:)
A case of PEBKAC.
